.circleci/config.yml

```
@@ -41,33 +41,19 @@ jobs:
                   exit 0
             fi
 
+            git config --global user.email '<>' # Leave email blank
+            git config --global user.name "Circle MonoRepo Automatic Deployer"
+
             for APP in $APPS_MODIFIED
             do
-              if [ "$APP" == "$CIRCLE_BRANCH" ]
+              if [ "$APP" != "$CIRCLE_BRANCH" ]
```

You can probably keep this the way it was: 

```suggestion
              if [ "$APP" == "$CIRCLE_BRANCH" ]
                  CREATE_APP=true ./deploy "$APP"
                  exit 0
```

---

I actually reversed this to make it break earlier. So the if guard has a continue inside, and then the deploy is de-dented.